.gitignore

```diff
@@ -1,11 +1,73 @@
+# Xcode
+#
+# gitignore contributors: remember to update Global/Xcode.gitignore, Objective-C.gitignore & Swift.gitignore
```

Comment:
_Side-note: we should also share our .gitignore across repos later (same idea as the rest, a gitignore in Eve as the source of truth being sync'd using a rake task that we'd run from time to time)_